pkg/runner/run_context.go

```diff
@@ -46,6 +46,11 @@ type RunContext struct {
 	Composite        *model.Action
 	Inputs           map[string]interface{}
 	Parent           *RunContext
+	masks            []string
```

Comment:
Please use  `Masks` as the name.
I use an external log formatter, which has no access to lowercase properties.

Reply:
done!